src/main/java/uk/gov/digital/ho/hocs/queue/common/QueueListener.java

```diff
@@ -32,7 +32,13 @@ public void onComplaintEvent(String message, @Header("MessageId") String message
                     log.warn("Message flagged to ignore: {}", messageId);
                 }
             }
+            if (messageHandler.getMessageType().equals(MessageTypes.MIGRATION)) {
```

Comment:
This does actually need to be it's own class - probably `MigrationQueueListener` that has a `@ConditionalOnProperty` annotation for migration. 

The reason is because you have no DLQ you need to use the default `deletionPolicy` which deletes messages if no redrive policy is specified.

Reply:
Fixed as suggested.
Added ConditionalOnProperty for QueueListener and MigrationQueueListener.
The "case-creator.mode" property set with value "migration" to use MigrationQueueListener for Case Migration. Otherwise QueueListener for Case Creator by default.